pokt-network/pocket
# [Telemetry] Add event metrics for HotPOKT

## Description


This PR adds event-metrics to the HotPOKT consensus implementation using the telemetry module.

## Changes
Please mark the options that are relevant.

- [x] Added instrumentation to record the reception of a hotpokt message as an event for each block, this event has the following attributes:
     - namespace `consensus`
     - event_name `hotpokt_messages`
     - labels: [height, step, validator_role]

- [x] Added logql query to process the recorded events and fed the results to a numerical series graph

## How Has This Been Tested?
__

1. Spin up the stack
```zsh
$ make compose_and_watch
```

2. Make sure you spin the client up, and that kick the blockchain off:
```zsh
$ make client_start && make client_connect
```

3. Choose:
```
✔ TogglePacemakerMode
✔ TriggerNextView
```

4. Open Grafana at [https://localhost:3000](https://localhost:3000)
5. Open the `HotPokt Telemetry Graphs` Dashboard

6. Verify the results on the dashboard

### Additional items for review

The query that was used to generate the graph is the following:
```logql
```

So I need you to also revise the logic in this query.


## Owners
@derrandz
